Resources/input/fsh/valueSets.fsh

```diff
@@ -39,3 +39,9 @@ Title: "ISiKAccountType"
 * $v3-ActCode#AMB "ambulatory"
 * $v3-ActCode#IMP "inpatient encounter"
 
+ValueSet: ISiKAccountIdentifierType
+Id: ISiKAccountIdentifierType
+Title: "ISiKAccountIdentifierType"
+* insert Meta
+* $v2-0203#AN "Account number"
```

Comment:
siehe Kommentar in https://github.com/gematik/spec-ISiK-Basismodul/pull/269/files#diff-0b9cf46d0a25828776f3d6aeca3367c5164e86302c5429ff7985e608cfb2fd92